CONTRIBUTING.md

```diff
@@ -321,6 +321,12 @@ development best practices to enhance your Drupal projects.
 that is downloaded when installing a site locally via Lando, or DDev.  See 
 "Visual Studio Code Integration"
 
+## Recommendation - macOS CLI Users
+Some patches do not apply via composer when running 'composer install' or 'composer update -W'. The solution is to install gpatch. 
```

Comment:
```suggestion
Some patches may not apply via composer when running `composer install` or `composer update` on macOS. This is because the version of the `patch` utility that is included with macOS differs from the version of GNU patch included with most Linux operating systems. A workaround for this problem is to install GNU patch using homebrew.
```